src/navigation/_CollapsingNavigationMenu.jsx

```diff
@@ -284,25 +296,32 @@ const CollapsingNavigationMenu = ({ selectedPath = undefined, menuItems, onSelec
     }
   };
 
-  const renderMenuItems = (currentMenuItem, parent = undefined, firstLevel = false) => {
+  const renderMenuItems = (currentMenuItem, parent = undefined, firstLevel = false, indexPath) => {
     if (!currentMenuItem) {
       return undefined;
     }
 
-    return currentMenuItem.map((item) => {
-      const id = item.name.split(' ').join('-');
-      const itemIsOpen = openKeys[item.path];
+    return currentMenuItem.map((item, index) => {
+      let currentPath;
+      if (!indexPath) {
+        currentPath = `idxPath-${index}`;
+      } else {
+        currentPath = `${indexPath}-${index}`;
+      }
+      const id = `${item.name.split(' ').join('-')}-${currentPath}`;
+      const itemIsOpen = !!openKeys[item.path];
       const isSelected = selectedPath === item.path;
 
       visibleNodes.push({ id, parent });
 
       return (
         <CollapsingNavigationMenuItem
-          item={item}
           id={id}
+          key={id}
```

Comment:
Will this also fix this issue I'm seeing when clicking nested items in nav?
![Screen Shot 2020-08-05 at 10 10 35 AM](https://user-images.githubusercontent.com/14099737/89430114-014ebd80-d704-11ea-87e6-07a190389c03.png)

Reply:
Yeah that is fixed